### ISSUE TITLE
Python version

### ISSUE THREAD

**Issue by Itsmmai** [linuxserver/docker-nzbget#91]
Hi for nzbget working with a script I need a newer python version.
Now it tells me: nzbToSickBeard: Please use a newer version of Python.
CUrrent Python version is 2.7 and I need at least 3.x can you update or tell me how to update?

Kind regards,
Abel

**Comment by matapala** [comment]
I just installed python3 within the container:
`docker run -it <name_of_your_nzbget_conatiner> ash`
once inside the container:
`apk update && apk add python3`
then link /usr/bin/python to /usr/bin/python3 ( check path before )

If you're not using docker volume, all changes will be lost on container restart.

**Comment by djderan** [comment]
This is what i did as a temporary work around.

docker exec -it $nzbget_container_name apk add python3
docker exec -it $nzbget_container_name ln -s /usr/bin/python3 /usr/bin/python --force

**Comment by CHBMB** [comment]
Python is a bit of a mess at the moment, some projects have been updated to python 3, some are still in the midst of migration, the approach we're currently taking is to include python2 and python3 if the underlying project and it's common plugins/scripts.

https://github.com/linuxserver/docker-nzbget/commit/c87dc842020ee2b0f018c6f16cf15b1ff9d41c53

Looks like nzbtomedia has done some backtracking from it's python2 to python3 migration.  New release 5 mins ago adds back python2 compatibility

https://github.com/clinton-hall/nzbToMedia/releases

So, bottom line, container already has python 3 now, and the scripts causing the issue have been reverted to continue to support python 2.

 So, if you wish to migrate to python 3 I wonder if the shell override is the way forward to avoid getting down and dirty with the terminal. 

![2020-01-04_09-54](https://user-images.githubusercontent.com/3609888/71763829-7e992200-2ed8-11ea-935e-7fac879243c6.png)

**Comment by djderan** [comment]
I can confirm that adding ".py=/usr/bin/python3" to the shelloverride option in extension scripts works.

**Comment by thelamer** [comment]
I guess we should add the override option in the readme so we can close out this issue.

**Comment by thelamer** [comment]
closing via https://github.com/linuxserver/docker-nzbget/pull/92
